### ISSUE TITLE
Automated Farmer quest# 939 unbuildable item LV Era

### ISSUE THREAD

**Issue by Obeliske1** [GTNewHorizons/GT-New-Horizons-Modpack#7457]
#### Which modpack version are you using?
1.7.10 the most recent release
#
#### If in multiplayer; On which server does this happen? 
Private server, confirmed in single player
#
#### What did you try to do, and what did you expect to happen?
Attempting to make an electric hoe requires a steel hoe head item number 6955/4305.
1, attempted to make it via tinkerers tools the part doesn't exist there. Checked the pattern, it's a cut pattern. Made the pattern placed it on the smeltery as there doesn't appear to be any other way to use these patterns. Made an extruder shape. Extruder at LV era is only capable of working rubber and (I think) plastic.
2, made a second pattern and attempted to use the smeltery to made a pattern in an attempt to make a steel one. This didn't work either
#
#### What happened instead? (Attach screenshots if needed)
1,Nothing as it literally can't be done
2,Also didn't work
After some research I was unable to find a steel hoe head in the NEI, I found reference on the Discord to the pattern being removed in 2017, not sure if this is still the case but cannot proceed with this. Somebody suggested waiting until MV age to do it with the extruder saying it may be completable then. Have not yet tested this via single player.
#
#### What do you suggest instead/what changes do you propose?
Either making the item creatable via the smeltery or changing the item in the recipe to something else a steel shovel head perhaps.

**Comment by Prometheus0000** [comment]
> 1.7.10 the most recent release

This means _nothing_.

> Automated Farmer quest# 939 unbuildable item LV Era

You didn't even say what it is.

As far as I know this is fixed/only not working on un-updated servers or something. But I can't search for the item if I don't know _what it is_.

**Comment by Obeliske1** [comment]
It's literally the first line in "What did you try to do".

I say the most recent release because there's a dingbat who argued with me on one of my posts because my fingers typed 1.1.70 and I don't want to have that conversation again. If it means nothing to you why bother responding to it. 

As for your claims of me not saying what it is how would you know if it's fixed or working if you don't know what it is.

**Comment by Prometheus0000** [comment]
> It's literally the first line in "What did you try to do".

Again, this doesn't help if I don't have the game open.

> I say the most recent release because there's a dingbat who argued with me on one of my posts because my fingers typed 1.1.70 and I don't want to have that conversation again. If it means nothing to you why bother responding to it.

People constantly say 'latest release' when it's not. The latest release is either the last one in the announcement channel, or one dream built for testing if there is one. And the previous is usually also for testing. The latest stable version is a different thing, and bugs are usually fixed in the actual latest version. Well, hopefully. And the 'latest' on technic or CF can be different too.

> As for your claims of me not saying what it is how would you know if it's fixed or working if you don't know what it is.

I recall one of the crop machine recipes being fixed, but I can't remember the name. And it's on you to report it properly. And try to use search to see if it was fixed already. There was a reasonably long discussion about it. Either here or in discord.

**Comment by Obeliske1** [comment]
I think maybe I'm not understanding what your referring to not helping. I'm not trying to be dense here (just naturally so I guess) but I state what item is not working properly, I state the quest, I give both the quest and the item IDs, I state that the item can't be built. What information are you looking for here that I'm not providing??? All you've said is I didn't say what it is or that it doesn't help. What did I not say that it is, where exactly am I missing information or what information that you need?

I appreciate your directing me to the announcements channel thank you, I was unaware that this was a thing and will be updating my server immediately. Or at least trying to figure out how to :P

I did use the Discord in an attempt to get some help/info but received none, and the last time I had an issue I was told to bring it here. So that's what I'm doing with an issue that I don't know more about.

**Comment by Prometheus0000** [comment]
> What information are you looking for here that I'm not providing

The name of the item. So I can search for the ticket for the same issue and tell you it's fixed (definitively). Because we already had a discussion about it not being craftable. The recipe was changed, but I'm not sure if the server files were updated correctly, which caused a server/client recipe desync. Which may or may not be fixed. Those are separate issues BTW.

Also, item IDs shift per world, much less per computer.

I remembered the name:
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7148
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7200

**Comment by Obeliske1** [comment]
Well hopefully following your earlier instructions on the announcements page will fix this issue, I will of course post results here. Thank you for the help.

**Comment by Prometheus0000** [comment]
The newer builds might break stuff (like the pyro, cracker, or ebf though), it might be safer just to copy the script file from the client over to the server (and take less time).

**Comment by Obeliske1** [comment]
Your killing me >.< I haven't run a MC server for almost a decade only just got back into it when I heard of this modpack and one of my friends refused to play on the online server because... well because he's crazy and thinks he's going to be griefed on a well moderated server. So I rented a server specifically for him. There's about 8 of us and I have absolutely no idea what I'm doing.

I don't suppose you'd know of a good Youtube video on how to do this? It's either learn or update the server :P Which would honestly make me sad as we JUST got out EBF up and running :P

**Comment by Prometheus0000** [comment]
Someone on discord would know, but replacing 1 file shouldn't be too hard? You can try the help channel.

**Comment by Obeliske1** [comment]
Oh and just cause I'm an idiot and rereading this 9 hours later it's the steel hoe head that can't be made, at least not in the LV era.

**Comment by Prometheus0000** [comment]
Since this is a duplicate of #7200, I'm gonna close it now, ok?

**Comment by Obeliske1** [comment]
This is for the Automated Harvest not the Crop Harvester. Sorry.

https://imgur.com/a/mJX8lSQ

As seen in the picture, steel hoe head is uncraftable for the quest "Automated Farmer"

You can make a pattern for the extruder but it's literally an uncraftable item, it doesn't exist in the NEI.

**Comment by Prometheus0000** [comment]
Ah, whoops?

**Comment by Prometheus0000** [comment]
![image](https://user-images.githubusercontent.com/8517039/107714233-e785c880-6c9a-11eb-9599-14421bed8a49.png)
![image](https://user-images.githubusercontent.com/8517039/107714337-18fe9400-6c9b-11eb-83ee-9d47c3bc2023.png)

This is why checking the actual latest is important, as it seems to already be fixed.
You need an at least partially filled re-battery too.

**Comment by Obeliske1** [comment]
Not arguing with that, hard to argue with proof :P I'm running 2.1.0.0 however so I assume this is fixed in a dev build? Sorry still learning about github and how it works, is there a way to see a list of things that have been changed in the soon to be released version so that I don't waste your time further on things like this?

**Comment by Prometheus0000** [comment]
I'm not sure when it was fixed.

> is there a way to see a list of things that have been changed in the soon to be released version so that I don't waste your time further on things like this

Sadly, no. You just gotta check the latest version, github (issues/prs/commits, and maybe discord. And/or download the latest version from the jenkins (for the built jars from the repos).

Just use admin powers to trade the ingredients for the items for now I'd say. It's not like you need more than 1 a player. And honestly, you don't need to hoe stuff _that_ much.

**Comment by Obeliske1** [comment]
100% agree and that's exactly what I'm going to do :) Only 8 of us anyway. Much appreciate the help though man thank you :)
